src/darker/__main__.py

```diff
@@ -380,7 +380,9 @@ def main(argv: List[str] = None) -> int:
         print("\n")
 
     if args.isort and not isort:
-        raise MissingPackageError(f"{ISORT_INSTRUCTION} to use the `--isort` option.")
+        raise MissingPackageError(
+            f"{get_extra_instruction('isort')} to use the `--isort` option."
+        )
```

Comment:
What's about this kind of code for `flynt` ?!?

Reply:
This has been now added for `flynt` as well.